_format/1.1/index.md

```diff
@@ -159,6 +159,7 @@ the client and represents a new resource to be created on the server.
 
 In addition, a resource object **MAY** contain any of these top-level members:
 
+* `revision`: a [revision string][revision] representing a specific version of an individual resource uniquely identified by `type` and `id`.
```

Comment:
this could also be a recommendation as a [resource meta](http://jsonapi.org/format/#document-resource-objects)